daemon/src/projection.rs

```diff
@@ -812,9 +815,18 @@ impl From<Order> for CfdOrder {
                 .try_into()
                 .expect("settlement_time_interval_hours is always positive number"),
             opening_fee: Some(order.opening_fee.to_inner()),
-            funding_rate_annualized_percent: AnnualisedFundingRate::from(order.funding_rate)
+            funding_rate_annualized_percent: AnnualisedFundingPercent::from(order.funding_rate)
                 .to_string(),
-            funding_rate_hourly_percent: HourlyFundingRate::from(order.funding_rate).to_string(),
+            funding_rate_hourly_percent: HourlyFundingPercent::from(order.funding_rate).to_string(),
+            initial_funding_fee_per_parcel: calculate_funding_fee(
+                order.price,
+                parcel_size,
+                order.leverage,
+                order.funding_rate,
+                SETTLEMENT_INTERVAL.whole_hours(),
+            )
+            .expect("to be able to calculate funding fee")
```

Comment:
Should probably use a `TryFrom` to avoid this expect.

Reply:
I added https://github.com/itchysats/itchysats/pull/1254/commits/55b7b1dca664fa1cdb5725321caa354f2ded0667 to achieve this. Probably would be even better to report failure back to the caller. I think it *can* happen that this fails - if it fails, it will fail on the maker and taker side, so the order would be invalid on both sides.